apps/backend/gateway/src/test/resources/bootstrap-test.yaml

```
@@ -4,6 +4,16 @@ spring:
+    hibernate:
+      ddl-auto: create-drop
 eureka:
   client:
     enabled: false
+jwt:
+  secret: ${JWT_SECRET}
+app:
+  permitted:
+    routes: /api/v1/user/login,/api/v1/user/register,/api/v1/user/verify,/csrf,/test-csrf,/actuator/*
+  access-token-expiration: 3600
+  refresh-token-expiration: 86400
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)